test/unit/event_store/repository_test.rb

```diff
@@ -104,7 +104,7 @@ def test_handle_invalid_event
     # it should notify bugsnag
     Bugsnag.expects(:notify).once
 
-    repository.publish_event(invalid_dummie_event)
+    refute repository.publish_event(invalid_dummie_event)
```

Comment:
In `master` branch with this test:
![image](https://user-images.githubusercontent.com/11318903/50002189-8a4d3a00-ff9f-11e8-8c46-cc84193e9aa1.png)

In this branch it passes 😄